container-specs.md

```diff
@@ -7,16 +7,16 @@ We explain here the containers metadata, their meaning and use in BioContainers:
-| base image            | The original image where the software has been built                                                                       | base image: biodckr/biodocker                            |
-| software version      | Every BioContainer should contain the actual version of the software and tool                                              | software version: 2015020                                |
-| software              | Name of the software                                                                                                       | software: Comet                                          |
-| description           | A short description of the software or tool.                                                                               | description: Peptide identification search tool          |
-| website               | The original software website where the user can find information about the tool the algorithms and examples.              | website: http://comet-ms.sourceforge.net/                |
-| documentation         | URL(s) containing information about how to use the software.                                                               | doc pages: http://comet-ms.sourceforge.net/              |
-| license               | SPDX license specification. If not in the SPDX list, then set short description and specify URL in license_file>                                                                                   | SPDX:Apache-2.0          |
-| license_file          | license path location in the container or url (according to license requirements |          |  
-| tags                  | Tags about the software that enable to find and classify the software tool.                                                | tags: proteomics, mass spectrometry, biocontainers       |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
+| base_image            | The original image where the software has been built | base_image: biodckr/biodocker |
+| software.version      | Version of the software and tool                     | software.version: 2015020     |
+| software              | Name of the software                                 | software: Comet               |
+| about.summary         | A short description of the software or tool.         | about.summary: Peptide identification|
+| about.home            | The original software website where  and examples.   | about.home: http://comet-ms.sourceforge.net/  |
+| about.documentation   | URL(s) containing information about software            | aobut.documentation: http://comet-ms.sourceforge.net/     |
```

Comment:
there is typo here (aobut)

Reply:
Typo has been fixed. I have also remove the CMD as suggested by @mr-c